api/build-with-serders.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Smile
+smile_latest_release=$(curl -s "https://api.github.com/repos/kafbat/ui-serde-smile/releases/latest")
```

Comment:
Can we collapse this script even more so we can have a list of repos (like kafbat/ui-serde-smile) so we could iterate over it and pull each jar?